### DISCUSSION TITLE
New Release Soon? 1.6.0?

### DISCUSSION THREAD

**Discussion by fawadasaurus** [grafana/tempo#1969] [General]
Hi.

I am wondering if we are going to have a new release soon?  The last release was in August and there have been many changes pushed.  Any update on when 1.6.0 will be released?

Thanks

**Comment by mapno** [comment]
Hi @fawadasaurus, we're aiming for a 2.0 release very soon (a few weeks?), that will include all the new changes: TraceQL, more stable vParquet, and many more. We don't have a more precise deadline at the moment.
